utilities/fuzzer.py

```diff
@@ -15,6 +15,29 @@
 
 logger = logging.getLogger()
 
+class FilePool(object):
+    """This implements a little pool of filenames, so we can reuse files instead of constantly
+    creating and deleting files.
+
+    We still create and delete trace-files, but this speed up test generation quite a lot even so 
+    """
+    def __init__(self):
+        self.counter = 0 
+        self.frees = []
+        self.locked = []
+    
+    def get(self):
+        if len(self.frees) > 0:
+            return self.frees.pop()
+        n = "pool_%d" % self.counter
+        self.counter = self.counter+1
+        return n
+
+    def put(self, f):
+        self.frees.append(f)
+
+
+fPool= FilePool()
```

Comment:
avoid globals. lets have this live inside a class. I'll manually merge this after #116

Reply:
Generally I agree, for pools however, might be ok to use globals